ibis/tests/expr/mocks.py

```diff
@@ -415,6 +413,18 @@ def compile(
         return queries[0] if len(queries) == 1 else queries
 
 
+def table_from_schema(name, meta, schema, *, database: str | None = None):
```

Comment:
This function was only moved, I don't plan on refactoring it here, or probably ever 😛